soname change 12:0:0 (.so.12.0.0) -> 12:0:1 (.so.11.1.0) looks like mistake... it was meant to be 12:1:0 (.so.12.0.1) or 13:0:1 (.so.12.1.0)
--- libosip2-5.1.1/configure.ac.orig	2020-01-21 17:42:22.616484653 +0100
+++ libosip2-5.1.1/configure.ac	2020-01-21 17:47:55.654680430 +0100
@@ -5,7 +5,7 @@
 AC_MSG_NOTICE([libosip2                The GNU Open SIP library.])
 AC_MSG_NOTICE([Copyright (C) 2001-2020 Aymeric MOIZARD - <amoizard@antisip.com>])
 
-AC_SUBST(LIBOSIP_SO_VERSION, 12:0:1)
+AC_SUBST(LIBOSIP_SO_VERSION, 12:1:0)
 AC_SUBST(OSIP_VERSION, $VERSION)
 
 AC_MSG_RESULT([Configuring ${PACKAGE} ${VERSION}])
